test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -34,6 +34,15 @@
     ('quantized::batch_norm', datetime.date(2020, 4, 20)),
     ('aten::sizes', datetime.date(2020, 4, 30)),
     ('aten::strides', datetime.date(2020, 4, 30)),
+    ('quantized::conv_prepack', datetime.date(2020, 6, 1)),
```

Comment:
what's going on here? why do you need to put it into BC test?

Reply:
Oh, this is mistake from when I typoed `conv_prepack` for an registration, and that lit up the bc check, and then when I fixed the typo I didn't fix the registration

Reply:
I need this in the BC test because I made a (technical) BC breaking change.

Previously, `conv_prepack` was registered with inferred schema:

```
-        .op("quantized::conv_prepack", // conv_prepack is deprecated, please use
-                                       // conv2d_prepack for 2D conv.
```

With this PR, it is now done with explicit schema:

```
+  // conv_prepack is deprecated, please use conv2d_prepack for 2D conv.
+  m.def("conv_prepack(Tensor weight, Tensor? bias, int[] stride, int[] padding, int[] dilation, int groups) -> Tensor");
```

Moving from implicit to explicit is considered bc breaking by the bc script, thus I must whitelist it.